assets/styles/banpage.less

```diff
@@ -0,0 +1,3 @@
+.ban-message {
+  min-height: 200px;
```

Comment:
Stick this to 8em, and it'll be the 8 rows you wanted, no matter what the lineheight is.